scIB/metrics/metrics.py

```diff
@@ -177,9 +184,10 @@ def metrics(
         n_isolated=None,
         graph_conn_=False,
         kBET_=False,
-        kBET_sub=0.5,
+        subsample=0.5,
```

Comment:
I thought this parameter was removed from the kBET method?

Reply:
this is only passed to LISI now?

Reply:
yes, that's why I renamed it. We never used it for kBET.